zhaquirks/xiaomi/aqara/sensor_switch_aq3.py

```diff
@@ -56,6 +57,12 @@
 }
 
 
+class OppleCluster(XiaomiAqaraE1Cluster):
+    """Opple cluster."""
+
+
+
+
```

Comment:
And remove some of these empty lines, so pre-commit is happy.
There should be two empty lines between class definitions.

Reply:
removed